COMPILING.md

```diff
@@ -77,16 +77,14 @@ We assume that you have a Debian/Ubuntu or Red Hat-like distribution.
 
 # COMPILATION ON SOLARIS 11
 
+We assume Solaris 11.4 or newer. You'll need to install Maven 3 manually.
```

Comment:
Hmm, the only other platforms to mention Maven are the Debian/Ubuntu/Fedora. It may be worth making clear that this is only required for JBMC? And, while at it, fix up the instructions for the other platforms?

Reply:
I can't seem to see those instructions for FreeBSD, MacOS, Windows?

Reply:
Fixed.